aztec/src/main/kotlin/org/wordpress/aztec/toolbar/ToolbarAction.kt

```diff
@@ -15,6 +15,12 @@ enum class ToolbarAction constructor(
         override val textFormats: Set<ITextFormat> = setOf())
     : IToolbarAction {
 
+    BACKGROUND(
```

Comment:
Hi @lvcasasanta, see point 3 in this comment: https://github.com/wordpress-mobile/AztecEditor-Android/pull/911#pullrequestreview-429501626. With the new background button, I think it makes sense to have it be easy for each project using Aztec to enable or disable the background button based off whether it is added as a plugin. @felipevaladares addressed this concern with this commit in the previous PR https://github.com/wordpress-mobile/AztecEditor-Android/pull/911/commits/a95d50e93e6fd0095b121c20fc7bb1389d6c24b1, but you are reversing that by re-adding the background button to the default Toolbar section here. 

I see this change was part of your [commit](https://github.com/wordpress-mobile/AztecEditor-Android/commit/aa138950e0541e9f8cf22fe035765a4d211cd896) for fixing the button active state issue. Can you fix that issue while maintaining out ability to not add the background button is the plugin is not added to our aztec instance? For reference you can check out the behavior of the Page and More buttons. If you comment out the [lines where they are added](https://github.com/wordpress-mobile/AztecEditor-Android/blob/a1386beed8d8215fe8ab79c501c1ec237ae8a785/app/src/main/kotlin/org/wordpress/aztec/demo/MainActivity.kt#L443), they no longer appear in the toolbar.

Let me know if that makes sense. Thanks!